### PR TITLE
Adding kabyle layout

### PR DESCRIPTION
**Pull request by BoFFire**
Adding kabyle layout as an azerty fork/style and some corrections via Weblate as a first shot. More improvements will come in the next version.

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -55,7 +53,7 @@
     <string name="custom_input_style_note_message">"Vous devez activer votre style de saisie personnalisé avant de l\'utiliser. Voulez-vous le faire maintenant ?"</string>
     <string name="enable">"Activer"</string>
     <string name="not_now">"Pas maintenant"</string>
-    <string name="custom_input_style_already_exists">"Le style de saisie suivant existe déjà : <xliff:g id="INPUT_STYLE_NAME">%s</xliff:g>."</string>
+    <string name="custom_input_style_already_exists">Le style de saisie suivant existe déjà : <xliff:g id="INPUT_STYLE_NAME" example="English (Dvorak)">%s</xliff:g></string>
```
Comment (rkkr) [inline app/src/main/res/values-fr/strings.xml:56]: Don't change the `<xliff:g id="INPUT_STYLE_NAME">%s</xliff:g>` part

File: app/src/main/res/values-kab/strings.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (rkkr) [inline app/src/main/res/values-kab/strings.xml:2]: It's not worth adding a translation file if you only translated 4 strings.

File: app/src/main/res/xml/kbd_kabyle.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<Keyboard
```
Comment (rkkr) [inline app/src/main/res/xml/kbd_kabyle.xml:21]: Keyboard layout not included in `res/xml/method.xml`. Have you even tested this?

File: app/src/main/res/xml/rowkeys_kabyle1.xml [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		<!-- -->
+        </case>
+        <default>
+            <Key
+                latin:keySpec="a"
+                latin:keyHintLabel="1"
+                latin:additionalMoreKeys="1"
+                latin:moreKeys="!text/morekeys_a" />
+            <Key
+                latin:keySpec="z"
```
Comment (rkkr) [inline app/src/main/res/xml/rowkeys_kabyle1.xml:73]: Do not redefine an AZERTY layout. Look how French language is implemented.

**Comment by BoFFire** [comment]
Thank you @rkkr I will submit another clean PR based on your comments, soon. Thank you very much for your advice <3.
Closed for cleaning.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (BoFFire) [reply app/src/main/res/values-fr/strings.xml:56]: Ok will add a "No translate" label in my Weblate.

File: app/src/main/res/xml/kbd_kabyle.xml (hunk 3 above)
Comment (BoFFire) [reply app/src/main/res/xml/kbd_kabyle.xml:21]: Ok thanks 👌

File: app/src/main/res/xml/rowkeys_kabyle1.xml (hunk 4 above)
Comment (BoFFire) [reply app/src/main/res/xml/rowkeys_kabyle1.xml:73]: Ok, nice !

**Pull request closed without merging**